source/features/linkify-code.tsx

```diff
@@ -63,6 +67,7 @@ void features.add(import.meta.url, {
 
 - Discussions: https://github.com/File-New-Project/EarTrumpet/discussions/877
 - Code Search: https://github.com/search?q=repo%3AKatsuteDev%2FBackground+marketplace&type=code
+- Code Search: https://github.com/search?q=org%3Arefined-github+%2F%23%5Cd%2B%2F&type=code
```

Comment:
Can you find a URL that has multiple results? We want to make sure this detection works even when there are multiple repos on the page. The wrong selector might pick the same/wrong repo for all results.

Reply:
https://github.com/search?q=%2F%23%5Cd%7B4%2C%7D%2F+language%3Atypescript&type=code

![image](https://github.com/refined-github/refined-github/assets/58778985/2fd65290-91e1-4120-80f6-a83e0a94e659)

![image](https://github.com/refined-github/refined-github/assets/58778985/a9bbd867-2256-40f0-9457-15015f994693)